airflow/providers/celery/provider.yaml

```diff
@@ -276,6 +276,20 @@ config:
         type: float
         example: ~
         default: "1.0"
+      task_acks_late:
+        description: |
+          If task runs exceed the visibility_timeout, Celery will re-assign new tasks, even if the
+          originally started task is still running successfully. The newly created task instance runs
+          then concurrently and the Airflow UI and logs only show an error message:
+          'Task Instance Not Running' FAILED: Task is in the running state
+          Setting task_acks_late to True will force Celery to wait until a task is finished before a
+          new task unstance is assigned. This effectively overrides the visibility timeout.
+          See also:
+          https://docs.celeryq.dev/en/stable/reference/celery.app.task.html#celery.app.task.Task.acks_late
+        version_added: ~
```

Comment:
Why no version specified?

Reply:
Copy and paste from another section, and the other examples I looked at did not have a version specified. Any suggestions?

Reply:
Should I change anything? Or can this conversation be considered resolved?

Reply:
Yes. add 2.9.0 - this is the first one new feature will appear.

Reply:
> Yes. add 2.9.0 - this is the first one new feature will appear.

Thanks, done in b3bf903

Reply:
I just reverted this change because it will be released with the celery provider and not the core release. Currently, we don't set the `version_added` for the providers' configurations.

Reply:
Heh... yeah provider. But we actually set the versions (and those are provider's versions).